launcher-package/src/universal/bin/sbt.bat

```diff
@@ -71,12 +71,17 @@ rem TODO: remove/deprecate sbtconfig.txt and parse the sbtopts files
 

```

Comment:
The line right above says
```
rem TODO: remove/deprecate sbtconfig.txt and parse the sbtopts files
```

I wonder if we should cast `sbtconfig.txt` to the abyss and only rely on `sbtopts` file, since https://github.com/sbt/sbt/pull/7966#issuecomment-2557559631 https://github.com/sbt/sbt/pull/7966#issuecomment-2557546813 are all caused by `sbtconfig.txt`.

Reply:
ok. It seems like `sbt.bat` is missing sbtopts mechanism, which virtualizes option parsing. I guess I'll revert the change related to `sbtconfig.txt`.